api/ocm/extensions/repositories/genericocireg/repository.go

```diff
@@ -44,23 +48,70 @@ type RepositoryImpl struct {
 	nonref   cpi.Repository
 	ocirepo  oci.Repository
 	readonly bool
+	// blobLimit is the size limit for layers maintained for the storage of localBlobs.
+	// The value -1 means an unconfigured value (a default from the blob limit configuration is used),
+	// a value == 0 disables the limiting and (a default from the blob limit configuration is ignored),
+	// a value > 0 enabled the usage of the specified size.
+	blobLimit int64
```

Comment:
Perhaps you should also use this description in line 66-69.

Reply:
done